relay_test.go

```diff
@@ -398,8 +398,7 @@ func TestLargeTimeoutsAreClamped(t *testing.T) {
 // TestRelayConcurrentCalls makes many concurrent calls and ensures that
 // we don't try to reuse any frames once they've been released.
 func TestRelayConcurrentCalls(t *testing.T) {
-	pool := NewProtectMemFramePool()
-	opts := testutils.NewOpts().SetRelayOnly().SetFramePool(pool)
+	opts := testutils.NewOpts().SetRelayOnly().SetCheckFrameLeaks()
```

Comment:
optional: what do you think about enabling the recording frame pool for all tests that are relay-only?

Reply:
i think we should, but there will likely be leaks that need to be fixed along with that, so prefer to defer that to the next change